### PR TITLE
Add secrets for IBM Z Ecosystem Cloud

### PR DESCRIPTION
**Pull request by ashfreeth**
This commit adds the secrets manifest file and directory for IBM Z Ecosystem Cloud, as part of enabling Operator certification for Z.

### CODE REVIEW

File: core-services/secrets/ibmz-eco/ibmz-eco-secrets.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# this is the Namespace in which your Secret will live
```
Comment (alvaroaleman) [inline core-services/secrets/ibmz-eco/ibmz-eco-secrets.yaml:1]: please add an OWNERS file

File: core-services/secrets/ibmz-eco/ibmz-eco-secrets.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  name: ibmz-eco
+---
+# the Group of people who should be able to manage this Secret
+kind: Group
+apiVersion: v1
+metadata:
+  name: ibmz-eco-admins
+users:
+  # these names are GitHub usernames
+  - ashfreeth
```
Comment (aosadchyy) [inline core-services/secrets/ibmz-eco/ibmz-eco-secrets.yaml:17]: Ash. Could you add me here too?   aosadchyy

File: core-services/secrets/ibmz-eco/ibmz-eco-secrets.yaml (hunk 1 above)
Comment (ashfreeth) [reply core-services/secrets/ibmz-eco/ibmz-eco-secrets.yaml:1]: Thanks, I've added it.

**Review by alvaroaleman — APPROVED** [review]
/label tide/merge-method-squash

**Pull request merged**